post-scan-actions/azure-python-slack-notification/template.json

```
@@ -0,0 +1,233 @@
+                        },
+                        {
+                            "name": "SUB_NAME",
+                            "value": "[variables('subscriptionName')]"
+                        },
+                        {
+                            "name": "BUS_TOPIC_CONNECTION_STRING",
+                            "value": "[listkeys(variables('scanResultAuthRuleResourceID'), variables('scanResultTopicAPIVersion')).primaryConnectionString]"
+                        }
+                    ]
+                },
+                "hostNameSslStates": [{
+                        "name": "[concat(variables('functionAppName'), '.azurewebsites.net')]",
+                        "sslState": "Disabled",
+                        "hostType": "Standard"
+                    },
+                    {
+                        "name": "[concat(variables('functionAppName'), '.scm.azurewebsites.net')]",
+                        "sslState": "Disabled",
+                        "hostType": "Repository"
+                    }
+                ]
+            }
+        },
+        {
+            "type": "microsoft.insights/components",
+            "apiVersion": "2020-02-02-preview",
+            "name": "[variables('applicationInsightsName')]",
+            "location": "[variables('location')]",
+            "kind": "web",
+            "tags": {
+                "[concat('hidden-link:', resourceId('Microsoft.Web/sites', variables('applicationInsightsName')))]": "Resource"
+            },
+            "properties": {
+                "Application_Type": "web",
+                "ApplicationId": "[variables('applicationInsightsName')]"
+            }
+        },
+        {
+            "apiVersion": "2019-05-01",
+            "name": "[variables('scanResultSubscriptionDeployment')]",
+            "type": "Microsoft.Resources/deployments",
+            "resourceGroup": "[variables('scanResultTopicResourceGroupName')]",
+            "subscriptionId": "[variables('scanResultTopicResourceSubscriptionID')]",
+            "properties": {
+                "mode": "Incremental",
+                "template": {
+                    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+                    "contentVersion": "1.0.0.0",
+                    "parameters": {},
+                    "variables": {},
+                    "resources": [{
+                        "type": "Microsoft.ServiceBus/namespaces/topics/subscriptions",
+                        "apiVersion": "[variables('scanResultTopicAPIVersion')]",
+                        "name": "[format('{0}/{1}/{2}', variables('scanResultTopicNamespace'), variables('scanResultTopicName'), variables('scanResultTopicSubscriptionName'))]",
+                        "properties": {
+                            "defaultMessageTimeToLive": "PT1H",
+                            "maxDeliveryCount": 3,
+                            "lockDuration": "PT4M"
```

Can be much shorter than quarantine function, which needs more time to move blobs.
```suggestion
                            "lockDuration": "PT1M"
```

---

changed to 1